INSTALLATION.md

```diff
@@ -62,4 +64,23 @@ Congratulations! You have successfully installed **MassaStation** on your comput
 6. The installation will begin, and you will see progress information in the terminal.
 7. Once the installation is complete, you can close the terminal.
 
-Congratulations! You have successfully installed **MassaStation** on your computer. You can start MassaStation by searching for it in the Applications folder.
+Congratulations! You have successfully installed **MassaStation** on your computer. You can start **MassaStation** by searching for it in the Applications folder.
+
+
+## Uninstallation
+
+### MacOS
+
+To uninstall **MassaStation** from your MacOS system, follow the steps below:
+
+1. Open the Terminal application on your MacOS system.
+2. Execute the following command in the terminal to download and run the **MassaStation** uninstaller script:
+
+   ```
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/massalabs/thyra/main/installer/macos/scripts/uninstall.sh)"
```

Comment:
warning /!\ the main branch will be the production branch for this script. 

we are used to publish releases for version management, this will bypass this.

I would recommend to add the uninstall script in the release assets.

Reply:
Good point, but let's come back to this later. I think the uninstallation process will be revisited as soon as we can to make it more sophisticated and so we could address this point at this moment. 
@gregLibert Do you agree?

Reply:
You can get the last tag using:
```shell
curl --silent "https://api.github.com/repos/massalabs/thyra/releases/latest" | jq -r .tag_name
```
or if you don't wnat to have jq dependency:
```shell
curl --silent "https://api.github.com/repos/massalabs/thyra/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'
```
once you have the last tag, you will be able to get the last version of the uninstaller.